packages/front-end/components/Experiment/NewExperimentForm.tsx

```diff
@@ -420,6 +445,11 @@ const NewExperimentForm: FC<NewExperimentFormProps> = ({
   const type = form.watch("type");
   const isBandit = type === "multi-armed-bandit";
 
+  const templateRequired =
```

Comment:
possibly should also check whether there are any valid templates first before forcing them to use it.